pkg/notifiers/sender.go

```diff
@@ -87,6 +104,96 @@ func TestPagerduty(key, msg string) error {
 	return nil
 }
 
+func TestWechat(secret, agent, corp, receiverType, receiver, msg string) error {
+	req, err := http.NewRequest(http.MethodGet, "https://qyapi.weixin.qq.com/cgi-bin/gettoken", nil)
+	if err != nil {
+		return err
+	}
+
+	q := req.URL.Query()
+	q.Add("corpid", corp)
+	q.Add("corpsecret", secret)
+	req.URL.RawQuery = q.Encode()
+
+	client := &http.Client{
+		Timeout: 15 * time.Second,
+	}
+	var wechatToken wechatToken
+	resp, err := client.Do(req)
+
```

Comment:
can we remove this blank line? The if err != nil will be the next line after you declared the err